Makefile

```diff
@@ -85,6 +85,7 @@ gettext:
 	go-xgettext -k=i18n.T "${JCLI_FILES}" > app/i18n/jcli.pot
 
 gen-data:
+# if you use MacOS and you don't install go-bindata, you can use command `brew install go-bindata` to install it
```

Comment:
See also [here](https://github.com/jenkins-zh/jenkins-cli/pull/263/files#diff-b67911656ef5d18c4ae36cb6741b7965R54). I'd prefer to add this comment into [CONTRIBUTING.md](https://github.com/jenkins-zh/jenkins-cli/blob/master/CONTRIBUTING.md) instead of here.

BTW, `you don't install go-bindata`. Really? I guess you want to say 'if you don't have go-bindata'.